Qdrant 1.7.0 release article (#438)
qdrant/landing_page: qdrant-landing/content/articles/qdrant-1.7.x.md

The latest version of your favorite vector search engine finally supports **sparse vectors**. That's the feature many of you requested, so why should we ignore it? 
We also decided to continue our journey with [vector similarity beyond search](/articles/vector-similarity-beyond-search/). The new Discovery API covers some utterly new use cases. We're more than excited to see what you will build with it! 
But there is more to it! Check out what's new in **Qdrant 1.7.0**!

1. Sparse vectors: do you want to use keyword-based search? Support for sparse vectors is finally here!
2. Discovery API: an entirely new way of using vectors for restricted search and exploration.
3. User-defined sharding: you can now decide which points should be stored on which shard.
4. Snapshot-based shard transfer: a new option for moving shards between nodes.

Do you see something missing? Your feedback drives the development of Qdrant, so do not hesitate to [join our Discord community](https://qdrant.to/discord) and help us build the best vector search engine out there!

## New features

Qdrant 1.7.0 brings a bunch of new features. Let's take a closer look at them!

### Sparse vectors

> The good old keyword-based search mechanisms are usually based on TFIDF, BM25, or similar algorithms. Internally, those methods also rely on vectors, but they are sparse. That means, **they contain a lot of zeros, and the number of non-zero values is usually small**.

Reviewer: The section on "Sparse vectors" starts with a casual tone ("The good old keyword-based search..."), which might slightly contrast with the more formal and technical style of other sections.

> Those sparse vectors are theoretically highly dimensional, definitely way higher than the dense vectors used in semantic search. However, since the majority of dimensions are usually zeros, we store them differently and just keep the non-zero dimensions. 

Reviewer 2: Suggestion : I think it should be "high dimensional" and not highly.

Until now, Qdrant has not been able to handle sparse vectors natively. Some were trying to convert them to dense vectors, but that was not the best solution or a suggested way. We even wrote a piece with [our thoughts on building a hybrid search](/articles/hybrid-search/), and we encouraged you to use a different tool for keyword lookup. 

> Things have changed since then, as so many of you wanted a single tool for sparse and dense vectors. That’s why we decided to implement native support for sparse vectors! Please wait closely for our next article, describing how to use this feature in practice. For now, you can check out the [sparse vectors documentation](/documentation/concepts/search/#sparse-vectors)!

Reviewer: ```suggestion
Things have changed since then. And responding to  [popular](https://github.com/qdrant/qdrant/issues/1678) [demand](https://github.com/qdrant/qdrant/issues/1135] , we've now introduced Sparse Vectors! 

Please wait closely for our next article, describing how to use this feature in practice. For now, you can check out the [sparse vectors documentation](/documentation/concepts/search/#sparse-vectors)!
```

There is a concept of `context`, which is a set of positive-negative pairs. Each pair divides the space into positive and negative zones. In that mode, the search operation prefers points based on how many positive zones they belong to (or how much they avoid negative zones).

The Discovery API can be used in two ways - either with or without the target point. The first case is called a **discovery search**, while the second is called a **context search**.

#### Discovery search

*Discovery search* is an operation that uses a target point to find the most relevant points in the collection, while performing the search in the preferred areas only. That is basically a search operation with more control over the search space.

![Discovery search visualization](/articles_data/qdrant-1.7.x/discovery-search.png)

Please refer to the [Discovery API documentation on discovery search](/documentation/concepts/explore/#discovery-search) for more details and the internal mechanics of the operation.

#### Context search

> The mode of *context search* is similar to the discovery search, but it does not use a target point. Instead, the `context` is used to navigate the HNSW graph towards preferred zones. It is expected that the results in that mode will be diverse, and not centered around one point.

Reviewer: Add link to HNSW graph?

```suggestion
The mode of *context search* is similar to the discovery search, but it does not use a target point. Instead, the `context` is used to navigate the [HNSW graph](https://arxiv.org/abs/1603.09320) towards preferred zones. It is expected that the results in that mode will be diverse, and not centered around one point.
```

> *Context search* might be an answer for those looking for some kind of exploration of the vector space. 

Reviewer 2: Suggestion to rewrite as : 
*Content Search* could serve as a solution for individuals seeking a more exploratory approach to navigate the vector space.

Reviewer 2: thank you !!  :)

![Context search visualization](/articles_data/qdrant-1.7.x/context-search.png)

### User-defined sharding

Qdrant's collections are divided into shards. A single **shard** is a self-contained store of points, which can be moved between nodes. Up till now, the points were distributed among shards by using a consistent hashing algorithm, so that shards were managing non-intersecting subsets of points.

> The latter one remains true, but now you can define your own sharding and decide which points should be stored on which shard. Sounds cool, right? But why would you need that? Well, there are multiple scenarios in which you may want to use custom sharding. For example, you may want to store some points on a dedicated node, or you may want to store points from the same user on the same shard.

Reviewer: ```suggestion
The latter one remains true, but now you can define your own sharding and decide which points should be stored on which shard. This functionality offers significant advantages in multiple scenarios in which you may want to use custom sharding. For example, you may want to store some points on a dedicated node, or you may want to store points from the same user on the same shard.
```

> The latter one remains true, but now you can define your own sharding and decide which points should be stored on which shard. Sounds cool, right? But why would you need that? Well, there are multiple scenarios in which you may want to use custom sharding. For example, you may want to store some points on a dedicated node, or you may want to store points from the same user on the same shard.

Author: That's a good question @atarora! @generall, What will be the preferred way of setting up multitenancy in Qdrant 1.7.0?

> While the old behavior is still the default one, you can now define the shards when you create a collection. Then, you can assign each point to a shard by providing a `shard_key` in the `upsert` operation. What's more, you can also search over the selected shards only by providing the `shard_key` parameter in the search operation.

Reviewer 2: While the 'existing' behavior is the default one. 
sounds more positive.
